Makefile

```diff
@@ -112,7 +112,11 @@ brew::
 	./scripts/brew.sh "${PROJECT}"
 
 .PHONY: lint_%
-lint:: .make/ensure/golangci-lint lint_golang
+lint:: .make/ensure/golangci-lint lint_golang lint_pulumi_json
```

Comment:
I don't think we currently run `make lint` at the top level in CI (at least from a quick look).  Would be nice to either do that (preferable imo, but not sure how hard it would be), or run the `lint_pulumi_json` target in CI separately.